python-lib/dku_aws/aws_command.py

```diff
@@ -28,8 +28,13 @@ def run(self):
         return (cmd, rv, o, e)
 
     def run_and_get_output(self):
-        return self.run()[2]
-    
+        result = self.run()
+        if result[1] != 0:
+            cmd = _convert_to_string(["aws"] + self.args)
+            raise Exception('Failed to execute command: \'%s\'.' % (' '.join(cmd)))
```

Comment:
If the error is too messy for this message that gets displayed on the UI, maybe you could put a "See log for more details" message, and then also log the error response with something like `logging.error`? Otherwise the user has no other option but to manually run the command to see what the error was (which we already have available to us).

We could also run the `eksctl` commands with `-v 0` or something so that it doesn't print a bunch of debugging stuff to standard error (and then we could put the standard error directly in the Exception message), but I think it's fine to just dump the entire error output to the log.

Reply:
Added the error output to the logs.